scripts/deploy_gke.sh

```diff
@@ -96,8 +96,8 @@ fi
 CLUSTER_NAME=$CLUSTER_PREFIX-$REGION
 
 # Deploy to GKE
-kustomize edit set image gcr.io/$IMAGE_PROJECT/datacommons-website=gcr.io/$IMAGE_PROJECT/datacommons-website:$WEBSITE_HASH
-kustomize edit set image gcr.io/$IMAGE_PROJECT/datacommons-nl=gcr.io/$IMAGE_PROJECT/datacommons-nl:$WEBSITE_HASH
+kustomize edit set image gcr.io/datcom-ci/datacommons-website=gcr.io/$IMAGE_PROJECT/datacommons-website:$WEBSITE_HASH
+kustomize edit set image gcr.io/datcom-ci/datacommons-nl=gcr.io/$IMAGE_PROJECT/datacommons-nl:$WEBSITE_HASH
```

Comment:
hm, so do we no longer need the IMAGE_PROJECT var set above?

Reply:
It's still used, but only in the replaced string. `kustomize edit set image <default> <new>` where the default on is always datcom-ci.